project/{{ cookiecutter.__folder_name }}/devops/README.md

```diff
@@ -51,6 +51,12 @@ STACK_NAME={{ cookiecutter.__devops_stack_name }}
 
 Note: The `.env` file is included in `.gitignore`, ensuring environment-specific configurations aren't pushed to the repository.
 
+
+### Server installation
+
+You need to Ubuntu or Debian in {{ cookiecutter.hostname }}, enable SSH, and install a supported version of Python 3 there.
```

Comment:
This still doesn't make sense:

"You need to Ubuntu or Debian in {{ cookiecutter.hostname }}"

Reply:
What about, `you need an Ubuntu or Debian based system in {{cookiecutter.hostname}}`

Reply:
```suggestion
You need either a Ubuntu or Debian based system for {{ cookiecutter.hostname }}, enable SSH, and install a supported version of Python 3 on that system.
```

Reply:
I think this does it. This is good.